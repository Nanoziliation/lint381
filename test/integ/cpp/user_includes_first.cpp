#include "baz.h"

#include <stdio.h>

#include "foo.h"
#include "bar.h"

// Malformed include for include parser testing.
#include <
