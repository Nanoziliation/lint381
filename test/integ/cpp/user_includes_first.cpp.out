user_includes_first.cpp:5:10: error: User include 'foo.h' should be before system includes
#include "foo.h"
         ^^^^^^^
user_includes_first.cpp:6:10: error: User include 'bar.h' should be before system includes
#include "bar.h"
         ^^^^^^^
